### ISSUE TITLE
[BUG] Need spacing between text and visibility label 

### ISSUE THREAD

**Issue by pylapp** [TwidereProject/TwidereX-iOS#129]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Given I want to write a new tweet
When I write text
Then I see the visibility label over the current text 

### Expected Behavior

More spacing is needed between input text and visibility label

### Steps To Reproduce

1. Tap to button to write a tweet
2. Write some text

See screenshot below

![47CBF1A4-7462-4F0A-B84C-5372E004ED7B](https://user-images.githubusercontent.com/7559007/210400295-086f0c06-cde1-44aa-b1cb-5bae4ca9623c.jpeg)

### Environment

```markdown
- Device: iPhone 14 Pro
- OS: iOS
- Version: 16.2
- Build: v1.4.3 (122)
- Domain: twitter.com
```

### Anything else?

Happy new year 🥳

**Comment by MainasuK** [comment]
This should be a UI bug. I'm going back to this project soon. 

Thank you point that out this.

**Comment by MainasuK** [comment]
Fixed in eeb4538.
